make/go/scripts/checknolintlint.bash

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+# Managed by makego. DO NOT EDIT.
+
+## checknolintlint exits with exit code 0 if nolintlint is enabled and configured according to standards.
+## Otherwise, it exits with exit code 1.
+
+set -euo pipefail
+
+if [[ ! -f .golangci.yml ]]; then
+    echo "nolintlint not enabled in .golangci.yml" >&2
```

Comment:
Clarifying my previous comment: my concern is if I don't have a `.golangci.yml`, then this script errors. In the context of `makego`, I think I should be able to not have a `.golangci.yml` file. I would instead exit 0 with no message.